tasks/install.yml

```diff
@@ -13,21 +13,27 @@
     list_files: yes
   register: file_list
 
-- name: Version name from folder path
+- name: Remove version name from folder path
   file:
     src: "{{ dokuwiki_install_path | dirname }}/{{ file_list['files'][0] }}"
     dest: "{{ dokuwiki_install_path }}"
     owner: www-data
     state: link
 
-- name: Install xml-php package used by dokuwiki.
+- name: Install php-xml and package used by dokuwiki.
   apt:
     name: "{{ item }}"
   with_items:
```

Comment:
why `with_items` if only one item is being installed?
Also these two tasks still exist in two files, please extract them into an own file and include them where needed (I think it would be once in `main.yml`?)